nvtabular/workflow.py

```diff
@@ -288,6 +288,13 @@ def _transform_ddf(ddf, column_groups):
 
     columns = list(flatten(cg.flattened_columns for cg in column_groups))
 
+    # Check if we are only selecting columns (no transforms).
+    # If so, we should perform column selection at the ddf level.
+    # Otherwise, Dask will not push the column selection into the
+    # IO function.
+    if all((c.op is None and not c.parents) for c in column_groups):
+        return _ddf_column_selection(ddf, column_groups)
```

Comment:
We've already gotten the names of all the columns on line 289 above - could we just go something like 
```suggestion
        return ddf[columns]
```
here instead?

Reply:
Ah - Good point!  Can't believe I missed that =)